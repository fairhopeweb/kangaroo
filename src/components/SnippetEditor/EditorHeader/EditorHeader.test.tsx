import { configure, shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import renderer from 'react-test-renderer';

import EditorHeader from './EditorHeader';

configure({ adapter: new Adapter() });

describe('<EditorHeader />', () => {
  const mockProps = {
    snippetId: 1,
    snippetTitle: 'test',
    onTitleChange: jest.fn(),
  };

  it('render without crashing', () => {
    shallow(<EditorHeader {...mockProps} />);
  });

  it('matches snapshot', () => {
    const wrapper = renderer.create(<EditorHeader {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
