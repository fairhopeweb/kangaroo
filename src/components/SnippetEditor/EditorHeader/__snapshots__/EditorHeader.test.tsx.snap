// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditorHeader /> matches snapshot 1`] = `
<div
  className="container"
>
  <h3
    className="bp3-heading"
  >
    <div
      className="bp3-editable-text snippetTitle"
      onFocus={[Function]}
    >
      <input
        className="bp3-editable-text-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Edit title..."
        type="text"
        value="test"
      />
    </div>
  </h3>
</div>
`;
